src/client.ts

```diff
@@ -3772,13 +3772,16 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         // If we expect that an event is part of a thread but is missing the relation
         // we need to add it manually, as well as the reply fallback
         if (threadId && !content["m.relates_to"]?.rel_type) {
+            const isReply = !!content["m.relates_to"]?.["m.in_reply_to"];
             content["m.relates_to"] = {
                 ...content["m.relates_to"],
                 "rel_type": THREAD_RELATION_TYPE.name,
                 "event_id": threadId,
+                // Set is_falling_back to true unless this is actually intended to be a reply
```

Comment:
```suggestion
                // Set is_falling_back to true unless this is actually intended to be a thread
```

because it's a reply unless it's a thread, essentially

Reply:
Reply is right here. The `is_falling_back` is to say, is the m.in_reply_to intentional, or just for the fallback

Reply:
conferred in DM, it is not backwards, spec just took some weird turns

Reply:
apparently we specified this field to go the inverse to what I thought, so the sentence is in fact correct.